maps/pom.xml

```diff
@@ -0,0 +1,33 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>maps</artifactId>
+    <packaging>jar</packaging>
+    <name>Gluon Maps</name>
+
+    <parent>
+        <groupId>com.gluonhq</groupId>
+        <artifactId>maps-parent</artifactId>
+        <version>2.0.0-ea+7-SNAPSHOT</version>
```

Comment:
if we want to be consistent with emoji pom.xml we should remove the parent pom here too?